### ISSUE TITLE
Resizing issues

### ISSUE THREAD

**Issue by keskinmbaha** [ashwinxkumar01/apps-college#10]
Home page has some weird resizing issues when the page is not full size. Picture is attached
<img width="841" alt="Screen Shot 2020-06-21 at 3 55 59 PM" src="https://user-images.githubusercontent.com/62824264/85236981-b9840900-b3d7-11ea-8101-40ca2497f49f.png">

**Comment by keskinmbaha** [comment]
Also not sure but the image on the login/sign up page becomes really narrow as well
<img width="841" alt="Screen Shot 2020-06-21 at 3 57 40 PM" src="https://user-images.githubusercontent.com/62824264/85237011-f4863c80-b3d7-11ea-98ae-db8e2729e709.png">

**Comment by keskinmbaha** [comment]
More resizing issues on the dashboard page
<img width="952" alt="Screen Shot 2020-06-21 at 4 00 20 PM" src="https://user-images.githubusercontent.com/62824264/85237064-55ae1000-b3d8-11ea-8c66-f4b401df23f5.png">

**Comment by amiteshksharma** [comment]
If it's only half of your screen, that would be around 7 to 8 inches, and would be more of an issue with formatting on mobile devices or tablets. With that being said, we are going to hopefully format it better for small devices.

**Comment by keskinmbaha** [comment]
Yeah just wanted to make sure you guys knew there were still resizing issues on basically all the pages.
